### DISCUSSION TITLE
Making test resources (src/test/resources) available under @QuarkusIntegrationTest ?

### DISCUSSION THREAD

**Discussion by nielsbasjes** [quarkusio/quarkus#26787] [Q&A] [answered]
Using Quarkus I have created a servlet (OpenAPI) with a few endpoints.
The functionality I'm exposing supports tweaking the functionality by adding additional config files which are read during startup.

In my src/test/resources I have added the appropriate files for these tests.
When I run my test in the normal way they all pass.
When I rerun the same tests with `@QuarkusIntegrationTest` the tests that rely on the presence of these resources fail.

I have tracked it down to the docker image that is run for these tests is essentially the same as the `production` image: it does not have the test resources in it (yes, I pulled the docker image apart to check all the files actually in there).

When I create an additional layer on top of this image with 
```dockerfile
FROM nielsbasjes/yauaa-quarkus:7.3.1-SNAPSHOT
COPY src/test/resources /app/resources/
```
and run that image then (although manually) is works as expected.

What is the correct way to make this work in a normal build?

[a comment by an automated account is not shown]

**Comment by nielsbasjes** [comment]
Note that my `quick check` by simply adding the resources to the docker image (as shown above) is in the general case incorrect as it is possible to have test resources that have been filtered by maven.

**Comment by geoand** [comment, marked as answer]
The idea of `@QuarkusIntegrationTest` is to run the production artifact and not something different, so this is something I don't think we'll do

**Reply by nielsbasjes** [reply]
Understood.
So I assume there is also no automated way to inject an additional layer on the docker image for these tests?

**Reply by geoand** [reply]
There is not
